http/headers/x-xss-protection.json

```
@@ -25,7 +27,8 @@
               "version_added": "8"
             },
             "opera": {
-              "version_added": true
+              "version_added": true,
+              "version_removed": "56"
```

Whoops!  Looks like you might have swapped the digits.  😛 

```suggestion
              "version_removed": "65"
```

---

Arh, dangit 😄